src/model/trust.page.model.ts

```diff
@@ -92,28 +92,30 @@ type TrustLegalEntityForm = {
 legalEntityId?: string;
 legalEntityName: string;
 roleWithinTrust: string;
-interestedPersonStartDateDay?: string
-interestedPersonStartDateMonth?: string
-interestedPersonStartDateYear?: string
-principal_address_property_name_number: string
-principal_address_line_1: string
-principal_address_line_2?: string
-principal_address_town: string
-principal_address_county: string
-principal_address_country: string
-principal_address_postcode: string
-service_address_property_name_number?: string
-service_address_line_1?: string
-service_address_line_2?: string
-service_address_town?: string
-service_address_county?: string
-service_address_country?: string
-service_address_postcode?: string
-governingLaw: string
-legalForm: string
-public_register_name?: string
-public_register_jurisdiction?: string
-registration_number?: string
+interestedPersonStartDateDay?: string;
+interestedPersonStartDateMonth?: string;
+interestedPersonStartDateYear?: string;
+principal_address_property_name_number: string;
+principal_address_line_1: string;
+principal_address_line_2?: string;
+principal_address_town: string;
+principal_address_county: string;
+principal_address_country: string;
+principal_address_postcode: string;
+service_address_property_name_number?: string;
+service_address_line_1?: string;
+service_address_line_2?: string;
+service_address_town?: string;
+service_address_county?: string;
+service_address_country?: string;
+service_address_postcode?: string;
+governingLaw: string;
+legalForm: string;
+public_register_name?: string;
+public_register_jurisdiction?: string;
+registration_number?: string;
+is_service_address_same_as_principal_address: yesNoResponse;
+is_on_register_in_country_formed_in: yesNoResponse;
 };
```

Comment:
Why don't need these changes/fields for the historic/former trustees. They are not present on the web page or the data model.
https://roe-v13.herokuapp.com/register/trusts/trust-historical-beneficial-owner

Reply:
In order to determine if the home address is same as the service address there is need to track the option clicked by the user in the page. same as the country of registration.

Reply:
Used as flags when radio buttons selected to track value

Reply:
<img width="384" alt="image" src="https://user-images.githubusercontent.com/59603637/219327382-2c561742-2799-4440-b27f-2a3bf70eb6f6.png">

Historical bo doesn't use the address fields or the register fields in the model and it is not captured on the web page. 

legal entity trustees capture both address types and the reg so i see the need there. Individual trustee capture both address type so I see the need there. Don't think we need it for former/historical.

https://roe-v13.herokuapp.com/register/trusts/trust-historical-beneficial-owner